### PR TITLE
immudb 0.6.0 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

High-speed immutable database 🔥

### CODE REVIEW

File: Formula/immudb.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    command = %W[
+      curl -s -XPOST -d '#{JSON.generate({ "key" => Base64.encode64("hello") })}'
+      http://localhost:3323/v1/immurestproxy/item/safe/get
+    ]
+    response = shell_output(command.join(" "))
+    assert_equal "world", Base64.decode64(JSON.parse(response)["value"])
+
+    assert_match "Uptime", shell_output("#{bin}/immuadmin stats -t")
+  ensure
+    pids.each do |pid|
```
Comment (SMillerDev) [inline Formula/immudb.rb:36]: No need to do this, brew will do this automatically these days

Comment (ankane) [reply Formula/immudb.rb]: Awesome!

File: Formula/immudb.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      exec bin/"immugw"
+    end
+    sleep 1
+
+    system bin/"immuclient", "safeset", "hello", "world"
+    assert_match "world", shell_output("#{bin}/immuclient safeget hello")
+
+    command = %W[
+      curl -s -XPOST -d '#{JSON.generate({ "key" => Base64.encode64("hello") })}'
+      http://localhost:3323/v1/immurestproxy/item/safe/get
```
Comment (SMillerDev) [inline Formula/immudb.rb:28]: You can't specify the port in config?

Comment (ankane) [reply Formula/immudb.rb]: 👍 added ports

**Review by SMillerDev — APPROVED** [review]
Thanks @ankane, you rock! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Review by chenrui333 — COMMENTED** [review]
Nice, got to know another database now.

**Pull request closed without merging**